src/spaces/openstreetmap.jl

```diff
@@ -1,32 +1,41 @@
+"""
+    OSM
+Submodule for functionality related to [`OpenStreetMapSpace`](@ref).
+See the docstring of the space for more info.
+"""
+module OSM # OpenStreetMap
+using Agents
+using OpenStreetMapX
+using LightGraphs
+using LinearAlgebra: dot
+
 export OpenStreetMapSpace, OSMSpace, TEST_MAP
-export osm_random_road_position,
-    osm_plan_route,
-    osm_map_coordinates,
-    osm_road_length,
-    osm_is_stationary,
-    osm_random_route!,
-    osm_latlon,
-    osm_intersection,
-    osm_road
-
-struct OpenStreetMapSpace <: DiscreteSpace
+export random_road_position,
+    plan_route,
+    map_coordinates,
+    road_length,
+    random_route!,
+    latlon,
+    intersection,
+    road
+
+struct OpenStreetMapSpace <: Agents.DiscreteSpace # TODO: Why is this a discrete space?
```

Comment:
Because it's graph space under the hood.

It's continuous since we build machinery around the OSMX.jl struct to make it so, but it still needs discrete methods for the most part.